### ISSUE TITLE
Fails tests without '.' in @INC

### ISSUE THREAD

**Issue by kentfredric** [miyagawa/cpanfile#47]
On 5.25.10 with -Ddefault_inc_excludes_dot

```
PERL_DL_NONLAZY=1 "/home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/bin/perl5.25.10" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
t/author-pod-syntax.t .. skipped: these tests are for testing by the author
Can't locate t/Utils.pm in @INC (you may need to install the t::Utils module) (@INC contains: /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10 /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10) at t/feature.t line 4.
BEGIN failed--compilation aborted at t/feature.t line 4.
t/feature.t ............ 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
Can't locate t/Utils.pm in @INC (you may need to install the t::Utils module) (@INC contains: /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10 /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10) at t/parse.t line 5.
BEGIN failed--compilation aborted at t/parse.t line 5.
t/parse.t .............. 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
Can't locate t/Utils.pm in @INC (you may need to install the t::Utils module) (@INC contains: /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10 /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10) at t/merge.t line 4.
BEGIN failed--compilation aborted at t/merge.t line 4.
t/merge.t .............. 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
Can't locate t/Utils.pm in @INC (you may need to install the t::Utils module) (@INC contains: /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10 /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10) at t/requirement.t line 4.
BEGIN failed--compilation aborted at t/requirement.t line 4.
t/requirement.t ........ 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
Can't locate t/Utils.pm in @INC (you may need to install the t::Utils module) (@INC contains: /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10 /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10) at t/mirror.t line 4.
BEGIN failed--compilation aborted at t/mirror.t line 4.
t/mirror.t ............. 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
Can't locate t/Utils.pm in @INC (you may need to install the t::Utils module) (@INC contains: /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/lib /home/kent/.cpanm/work/1488176173.29222/Module-CPANfile-1.1002/blib/arch /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10 /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10) at t/from_prereqs.t line 5.
BEGIN failed--compilation aborted at t/from_prereqs.t line 5.
t/from_prereqs.t ....... 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 

Test Summary Report
-------------------
t/feature.t          (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: No plan found in TAP output
t/parse.t            (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: No plan found in TAP output
t/merge.t            (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: No plan found in TAP output
t/requirement.t      (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: No plan found in TAP output
t/mirror.t           (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: No plan found in TAP output
t/from_prereqs.t     (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: No plan found in TAP output
Files=7, Tests=0,  0 wallclock secs ( 0.03 usr  0.02 sys +  0.56 cusr  0.12 csys =  0.73 CPU)
Result: FAIL
Failed 6/7 test programs. 0/0 subtests failed.
```

**Comment by jkeenan** [comment]
Please review this pull request:  https://github.com/miyagawa/cpanfile/pull/49

As there may be other CPAN distributions and darkpan code which uses 'cpanfile', an early release to CPAN (assuming p.r. is correct) would be greatly appreciated.

Thank you very much.
Jim Keenan

**Comment by karenetheridge** [comment]
perl 5.26.0 is now out; this issue manifests there.

**Comment by miyagawa** [comment]
Fixed in 1.1004
